### PR TITLE
bear stuff

### PR DESCRIPTION
**Pull request by AbdealiJK**

### CODE REVIEW

File: bears/python/PyImportSortBear.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 class PyImportSortBear(LocalBear):
 
+    LANGUAGES = ("Python",)
```
Comment (sils1297) [inline bears/python/PyImportSortBear.py]: do you want to update the coala requirement and use a string?

File: bears/python/PyImportSortBear.py [hunk 2]
```diff
@@ -20,7 +20,12 @@ def run(self,
             sort_imports_by_length: bool=False,
             isort_multi_line_output: int=4):
         """
-        Sorts imports for python.
+        Handle imports for python using the utility ``isort``. It can sort the
+        imports, segregate it into various sections, and also add comments
+        on top of each import section based on the configurations provided.
```
Comment (sils1297) [inline bears/python/PyImportSortBear.py]: remove the first sentence from this paragraph

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/364/commits/f59ca1d78a447ceccdc641009edb9bb307cbbc95 https://github.com/coala-analyzer/coala-bears/pull/364/commits/2a729e4b45c0713db14f7362f333f91f75a0f23d https://github.com/coala-analyzer/coala-bears/pull/364/commits/0114b627f73b2f021ea328ad46762d813ae9f72d

File: bears/coffee_script/CoffeeLintBear.py [hunk 3]
```diff
@@ -23,7 +23,10 @@ def convert_if_not_empty(value: str, conversion):
 @linter(executable='coffeelint')
 class CoffeeLintBear:
     """
-    Coffeelint's your files!
+    Use ``coffeelint`` to parse CoffeeScript code to help keep the code
+    clean and consistent and raise style inconsistencies.
```
Comment (sils1297) [inline bears/coffee_script/CoffeeLintBear.py]: how about

```
Check CoffeeScript code for a clean and consistent style.
```

shorter and it's the info the user actually wants IMO, keep the for more info paragraph

File: bears/coffee_script/CoffeeLintBear.py [hunk 4]
```diff
@@ -36,19 +21,19 @@ class CoffeeLintBear:
...
-                    message=row['message'],
-                    file=filename,
-                    line=convert_if_not_empty(row['lineNumber'], int),
-                    end_line=convert_if_not_empty(row['lineNumberEnd'], int),
-                    severity=self.severity_map[row['level']])
-            except KeyError:  # Invalid CSV line, ignore
-                pass
+        output = json.loads(output)
+
+        assert len(output) == 1  # More than 1 file parsed, something went wrong
```
Comment (sils1297) [inline bears/coffee_script/CoffeeLintBear.py]: why not make a comma instead of # and make your coment a failure message?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/364/commits/80da257a8c72d95047ac59996f5c87ca7e044312

File: bears/js/JSHintBear.py [hunk 5]
```diff
@@ -21,407 +18,357 @@ def bool_or_int(value):
...
+        output_regex=r'.+?: line (?P<line>\d+), col (?P<column>\d+), '
+                     r'(?P<message>.+) \((?P<severity>[EWI])\d+\)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL,
+                      'I': RESULT_SEVERITY.INFO})
+class JSHintBear:
+    """
+    Checks the code with jshint. This will run jshint over each file
+    separately.
+    """
```
Comment (rubdos) [inline bears/js/JSHintBear.py]: Compare with

> Check CoffeeScript code for a clean and consistent style.
> 
> For more information about coffeelint, visit <http://www.coffeelint.org/>.

Perhaps unify this sort of descriptions?

I suggest something along these lines:


> Detect errors and potential problems in JavaScript code.
> 
> For more information about jshint, visit <http://jshint.com/>.

This also gives an indication of what the bear does: linting vs style checking (cfr. CoffeeLint).

**Comment by rubdos** [comment]
ack 6c32a6a2d080feb4fc30bda2d4eaa887295bd3ec

**Comment by rubdos** [comment]
reack
a30be2c JSHintBear: Remove deprecated `es3` config
f9892a6 bear/*: Use string in LANGUAGES
6c32a6a JSHintBear: Add better documentation
7ad1366 JSHintBear: Add LANGUAGES
fe3b309 JSHintBearTest: Fix indentation
ede8a07 JSHintBearTest: Stop using prepare_file()
7d27b79 JSHintBear: Refactor to Linter v2 API
c405cd6 JSHintBear: Use new documentation style
390ed49 CoffeeLintBear: Use --format=raw for json
335a4d5 CoffeeLintBear: Provide better description
08c61ab CoffeeLintBear: Add LANGUAGES
fb12419 PyImportSortBear: Add more configs
e769cab PyImportSortBear: Improve documentation
aad58e8 PyImportSortBear: Add LANGUAGES
3218ec0 requirements: Update coala to latest dev release
2a729e4 PyImportSortBear: Use new documentation format
f59ca1d .coafile: Ignore links in LineLength

**Comment by AbdealiJK** [comment]
reack 5f6d160 63e1672

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/364/commits/109bd6dd1466feadf13591a035a61eb37cc0dfe7

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/364/commits/d699660ef133e87216c40e86951190019ea118ed

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**